server/webhook_jira.go

```diff
@@ -191,11 +192,12 @@ func mdUser(user *jira.User) string {
 }
 
 func truncate(s string, max int) string {
-	if len(s) <= max || max < 0 {
+	if utf8.RuneCountInString(s) <= max || max < 0 {
 		return s
 	}
+	runes := []rune(s)
 	if max > 3 {
-		return s[:max-3] + "..."
+		return string(runes[:max-3]) + "..."
```

Comment:
we can do operation string(runes) once above this condition

Reply:
Discussed